IPython Parallel installs just fine without jupyterlab, and the Void package
will add some version conflicts that prevent building.

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,6 +1,5 @@
 [build-system]
 requires = [
-  "jupyterlab>=3.0.0,==3.*",
   "hatchling>=0.25"
 ]
 build-backend = "hatchling.build"
